perllib/FixMyStreet/App/Controller/Waste.pm

```diff
@@ -936,6 +937,12 @@ sub get_original_sub : Private {
         });
     }
 
+    if ( $c->stash->{report} ) {
+        $p = $p->search({
+            id => { '<>' => $c->stash->{report}->id }
```

Comment:
Hmmm, it looks like this could only ever happen if you are a staff user in which case you the payment method is moot so this is probably not necessary.